credentials/src/main/kotlin/SSI.kt

```
@@ -182,6 +115,17 @@ public class VerifiableCredential private constructor() {
   }
 }
 
+private fun JWSAlgorithm.toVerifier(publicKeyJWK: JWK): JWSVerifier? {
+  return when (this) {
+    JWSAlgorithm.EdDSA -> Ed25519Verifier(publicKeyJWK.toOctetKeyPair())
+    JWSAlgorithm.ES256K -> ECDSAVerifier(publicKeyJWK.toECKey())
+    JWSAlgorithm.ES256 -> ECDSAVerifier(publicKeyJWK.toECKey())
+    JWSAlgorithm.ES384 -> ECDSAVerifier(publicKeyJWK.toECKey())
+    JWSAlgorithm.ES512 -> ECDSAVerifier(publicKeyJWK.toECKey())
+    else -> null
```

should we throw an exception ?